phys2bids/tests/test_integration.py

```diff
@@ -233,3 +233,54 @@ def test_integration_multirun(skip_integration, multi_run_file):
     # for run in ['1', '2']:
     #     assert isfile(join(conversion_path, f'Test2_samefreq_TWOscans_{run}_trigger_time.png'))
     assert isfile(join(conversion_path, 'Test2_samefreq_TWOscans.png'))
+
+
+def test_integration_matlab(skip_integration, matlab_file):
+
+    if skip_integration:
```

Comment:
Asking just because I'm seeing it here: why are we going for this approach when we could just tell pytest to run every test _but_ test_integration.py?